File: backend/src/main/java/top/vuhe/SmartWarningApplication.java
package top.vuhe;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

/**
 * @author zhuhe
 */
@EnableAsync
@SpringBootApplication
public class SmartWarningApplication {

    public static void main(String[] args) {
        SpringApplication.run(SmartWarningApplication.class, args);
    }

}

File: backend-kt/src/main/kotlin/top/vuhe/sw/SmartWarningApplication.kt
package top.vuhe.sw

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync

@EnableAsync
@SpringBootApplication
class SmartWarningApplication

fun main(args: Array<String>) {
    runApplication<SmartWarningApplication>(*args)
}
